### DISCUSSION TITLE
be 集群节点未知原因挂掉

### DISCUSSION THREAD

**Discussion by qinxiangyangnet** [apache/doris#28867] [RFC]
版本2.0.2，一个fe节点3个be节点，执行个查询操作，有一个be节点启动不起来，be.out
提示
![image](https://github.com/apache/doris/assets/50295203/b115d20b-1380-4a6f-b4dd-ef9acf089e7b)

**Comment by qinxiangyangnet** [comment]
be.warning 提示
![image](https://github.com/apache/doris/assets/50295203/2857d737-9cb9-4844-8121-d4fa22caeaad)

**Comment by qinxiangyangnet** [comment]
看网上解决方法说，查询列和数据库定义列不一样，找到那个查询sql就可以了，节点连启动都起不来，我怎么排查执行sql，而且sql那么多

**Comment by yagagagaga** [comment]
This bug has been fixed in version 2.0.3, and you can upgrade to the latest stable version of 2.0.
